src/clock.rs

```diff
@@ -295,7 +288,7 @@ pub struct ClockConfig<MCLK, SMCLK> {
 
 macro_rules! make_clkconf {
```

Comment:
Is this macro really necessary?

Reply:
Maybe not. The clock module has not yet been refactored.